src/Xiaomi_Scale.py

```diff
@@ -113,32 +108,10 @@ def handleDiscovery(self, dev, isNewDev, isNewData):
                             self._publish(round(measured, 2), unit, str(datetime.today().strftime('%Y-%m-%d-%H:%M:%S')), hasImpedance, miimpedance)
                             OLD_MEASURE = round(measured, 2) + int(miimpedance)
 
-            if not dev.scanData:
+            else:
```

Comment:
reviewing it I would argue we should actually remove lines 11 & 112 as they bring new value

Reply:
sorry, i don't understand this

Reply:
Typo. We should remove them as they bring **no** value